packages/text-input/_autosuggest.scss

```
@@ -34,6 +40,8 @@
     --jkl-autosuggest-controller-button-padding: 0;
     --jkl-autosuggest-controller-button-icon-size: #{jkl.rem(18px)};
     --jkl-autosuggest-item-margin: 0 0 #{jkl.rem(1px)} 0;
+    --jkl-autosuggest-no-hits-message-font-size: var(--jkl-small-font-size);
+    --jkl-autosuggest-option-padding: #{jkl.$spacing-xs};
```

Det mangler en kompakt versjon av variabelen som setter høyden til options. Foreslår å rename den til `...-option-height` også, siden vi ikke bruker den her til å styre høyden til noe inputfelt.

```suggestion
    --jkl-autosuggest-option-padding: #{jkl.$spacing-xs};
    --jkl-autosuggest-option-height: #{jkl.rem(32px)};
```